components/illustration.jsx

```diff
@@ -3,34 +3,29 @@ var ImageTag = require('./imagetag.jsx');
 
 var Illustration = React.createClass({
   propTypes: {
-    verticalLayout: React.PropTypes.bool,
     width: React.PropTypes.number.isRequired,
     height: React.PropTypes.number.isRequired,
-    className: React.PropTypes.string,
     src1x: React.PropTypes.string.isRequired,
     src2x: React.PropTypes.string,
     alt: React.PropTypes.string.isRequired,
     link: React.PropTypes.string
   },
   render: function() {
-    var classes = this.props.verticalLayout ? "illustration vertical-layout" : "illustration";
+    var classes = this.props.className ? ("illustration " + this.props.className) : "illustration";
     var image = <ImageTag width={this.props.width}
                           height={this.props.height}
-                          className={this.props.className}
```

Comment:
We shouldn't be doing this.  `className` should go to the component's wrapper instead.